common/src/main/java/com/netflix/conductor/common/run/Workflow.java

```diff
@@ -492,6 +502,12 @@ public Workflow copy() {
 		copy.setTasks(tasks.stream()
 				.map(Task::deepCopy)
 				.collect(Collectors.toList()));
+		copy.setEndTime(endTime);
```

Comment:
Wondering if adding these fields would cause any side effects, as the copy is being used in decider evaluations at different stages. Do we have enough test coverage around usage of copy() ?

Reply:
Verified that the integration/unit tests do cover these.